src/components/List/List.js

```diff
@@ -8,22 +8,27 @@ import './list.scss';
 @skinnable()
 @props({
   list: t.maybe(t.Array),
-  loading: t.Boolean
+  loading: t.Boolean,
+  error: t.maybe(t.Object)
 })
 export default class List extends React.Component {
 
   getLocals() {
+
     const {
       list,
+      error,
       loading
     } = this.props;
-
     return {
       list,
       loading,
-      shouldRenderPlaceholder: typeof list === 'undefined',
+      errorStatus: typeof error !== 'undefined' ? error.status: error,
+      errorMessage: typeof error !== 'undefined' ? error.data.message: error,
+      shouldRenderPlaceholder: typeof list === 'undefined' && typeof error === 'undefined',
```

Comment:
I don't really like using `typeof`, couldn't you get the same result by doing `!undefined` ?
If you want to be sure it's not another falsy value (like `''` or simply `false`) feel free to use `t.Nil.is()` from `tcomb`
you can see the full documentation [here](https://github.com/gcanti/tcomb/blob/master/docs/API.md)